xml/templates/message_templates/membership_online_receipt_text.tpl

```diff
@@ -154,14 +154,14 @@
 {/foreach}
 {/if}
 
-{if !( $contributeMode eq 'notify' OR $contributeMode eq 'directIPN' ) and $is_monetary}
+{if ($is_pay_later || $billingName)}
```

Comment:
we need to reverse the if order on this one

Reply:
why would that help with this if tho, this is the if saying are we printing either the Registered Email or Billing Name sections. Then we do or are you suggesting we completely get rid of the outer if all together?

Reply:
Right - the outer if is not required we want

{if $billingName}
.... show billing name & email
{elseif $email}
  ... show email
{/if}

I actually already did this on the same receipt but the html version